api/client/trust.go

```diff
@@ -37,7 +37,10 @@ import (
 	"github.com/docker/notary/tuf/data"
 )
 
-var untrusted bool
+var (
+	releasesRole = data.CanonicalTargetsRole + "/releases"
```

Comment:
`path.Join()` should always use a slash; https://golang.org/src/path/path.go?s=3776:3808#L140, whereas `filepath.Join()` would convert to a platform-specific separator; https://golang.org/src/path/filepath/path.go?s=5823:5855#L8